packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -598,9 +604,11 @@ const Summary: React.FunctionComponent = () => {
 		/>
 	);
 
+	const isPlansStepHighlighted = !! hasSelectedDomain || !! selectedPlan;
```

Comment:
I think we need the entire logic to highlight domain as a pre-existing condition here (including title) in order to prevent this case:
<img width="562" alt="Screenshot 2020-12-03 at 16 16 17" src="https://user-images.githubusercontent.com/14192054/101034653-37516080-3583-11eb-835a-f0b3846d02f8.png">

So I'm thinking of using something like this: `isDomainStepHighlighted && !! selectedPlan`

Reply:
Me and @ciampo have just gone through these two scenarios. It's an edge case as it stands but we agreed that the current step should only be highlighted if the previous step has been completed, and not base it on if _all_ previous steps.

This means that the user can only highlight the plans step if they select a domain (which they can only select if they've entered a valid title). If the user then decides to go back and change their title to the default title it doesn't seem right to then to 'disable' all the steps.

Reply:
In this case user had a default Site title (like all users coming from `/start`) and Premium plan pre-selected (coming from cart; see https://github.com/Automattic/wp-calypso/pull/47990)

Reply:
@razvanpapadopol I've implemented your suggested changes. Worth noting now that in this case a user had a default Site title and domain plan pre-selected (coming from cart) then the plans step will not be highlighted.

Reply:
🤦 then leave it as it was? https://github.com/Automattic/wp-calypso/pull/47503/commits/856bb448254629add939d9454bc9dec3b508e168#diff-bae2915b59f2e567e179b1ed48bdcb4d210fbc99442732fa6252d32ef7c2c181L603

Reply:
Having a domain from cart but not a plan is very unlikely scenario, so it's probably the lesser evil in this situation.

We should still add these notes to a separate issue, to discuss a final approach that aims at covering all edge cases

Reply:
Then there's the first scenario that if they have both a plan and a domain from cart, the plans step will be disabled because the title is still the default Site Title.

Reply:
I have reverted this change so the user must have a selected domain or a selected plan for the plans step to be highlighted.